File: app/src/main/java/com/juawapps/newsspread/categories/NewsCategoriesManager.java
package com.juawapps.newsspread.categories;


import java.util.ArrayList;
import java.util.List;

/**
 * News categories manager.
 * <p>
 * This class only gets the default categories but in the future is to have the categories
 * customizable.
 */

public class NewsCategoriesManager {

    public List<NewsCategory> getCategories() {
        return new ArrayList<>(DefaultNewsCategories.getCategories());}

}

File: app/src/main/java/com/juawapps/newsspread/categories/NewsCategoriesManager.kt
package com.juawapps.newsspread.categories


import java.util.ArrayList

/**
 * News categories manager.
 *
 *
 * This class only gets the default categories but in the future is to have the categories
 * customizable.
 */

class NewsCategoriesManager {

    val categories: List<NewsCategory>
        get() = ArrayList(DefaultNewsCategories.categories)

}
